### PR TITLE
Workshop Modification - WMS ID: 11514

### PR DESCRIPTION
**Pull request by essanhaji**
# WMS ID : 11514

# General requirements
1. Information in the workshop is adequate and updated
2. Code is correct and working
3. Links are correct
4. Make sure you enter the help email link in your manifest.json
5. Please make sure WMS URLs are updated as needed after your PR is approved

# Checklist - Refer to the QA document for the complete list 
Please confirm that the following is completed before submitting your PR

- [x] All filenames are lower case (including folders, images, files, etc.)
- [x] Filenames are descriptive
- [x] Your workshop folder structure should be similar to the one used in the sample workshop (https://github.com/oracle-livelabs/common/tree/main/sample-livelabs-templates/sample-workshop)
- [x] Are you using multiple versions (desktop/, sandbox/, tenancy/)? Make sure that each of them contains a manifest.json and an index.html
- [x] Image references in markdown contain an alternate text

### CODE REVIEW

File: managing-credentials-using-oci-vault/setup-oci-vault/oci-vault-setup-sandbox.md
Comment (denisemyrick) [inline managing-credentials-using-oci-vault/setup-oci-vault/oci-vault-setup-sandbox.md:1]: Verify your note syntax. It should be > **Note

Comment (denisemyrick) [reply managing-credentials-using-oci-vault/setup-oci-vault/oci-vault-setup-sandbox.md:1]: Verify your step numbering. Each task should start back at 1. In task 1 you go from step 3 to step 5 and then 18

Comment (denisemyrick) [reply managing-credentials-using-oci-vault/setup-oci-vault/oci-vault-setup-sandbox.md:1]: Also I noted that you have .mp4 files. We don’t allow uploads of videos into our repos. Link your YouTube video as recommended in the [step by step guide](https://oracle-livelabs.github.io/common/sample-livelabs-templates/create-labs/labs/workshops/livelabs/?lab=4-labs-markdown-develop-content#Task7:AddandembedaVideo)

Comment (essanhaji) [reply managing-credentials-using-oci-vault/setup-oci-vault/oci-vault-setup-sandbox.md:1]: The numbering looks fine to me, I double check it, would be nice if you could attach a screenshot
Thanks

Comment (essanhaji) [reply managing-credentials-using-oci-vault/setup-oci-vault/oci-vault-setup-sandbox.md:1]: The videos are removed as a part of this PR and replaced with YouTube links
Thanks

Comment (denisemyrick) [reply managing-credentials-using-oci-vault/setup-oci-vault/oci-vault-setup-sandbox.md:1]: You're right! I shouldn't do reviews pre-coffee. I'll double check now and approve.

**Review by denisemyrick — APPROVED** [review]

**Pull request merged**